src/main/java/org/folio/circulation/services/agedtolost/MarkOverdueLoansAsAgedLostService.java

```diff
@@ -113,6 +122,7 @@ private boolean shouldAgeLoanToLost(Loan loan) {
     return statusQuery.combine(dueDateQuery, CqlQuery::and)
       .combine(claimedReturnedQuery, CqlQuery::and)
       .combine(agedToLostQuery, CqlQuery::and)
-      .map(query -> query.sortBy(ascending("dueDate")));
+      .map(query -> query.sortBy(ascending("dueDate")))
+      .value();
```

Comment:
What happens if mapping the value fails?

Reply:
As far as I see the result can be failed only in `org.folio.circulation.support.http.client.CqlQuery#exactMatchAny` when array of values is empty. Should I handle the result fail case as well?

Reply:
Ok. I was mostly curious about why this code assumes the result succeeds rather than composing results together. 

I know that sometimes that isn't easy to do and if this one of those circumstances, that's ok, please resolve this comment.

Reply:
No prob, I have updated the code to use result for the queries.